jobbergateapi2/tests/apps/applications/test_routers.py

```diff
@@ -218,7 +218,7 @@ async def test_get_application_by_id_invalid(client, user_data, application_data
 @database.transaction(force_rollback=True)
 async def test_get_application_all_from_user(client, user_data, application_data):
     """
-    Return all applications from the user
+    Return all applications from the user.
```

Comment:
"""
Test that the correct application is returned for the user that created it.

This test proves that GET /application/<id> returns the correct application for the user making
the request. We show this ​by asserting that the application data in the response is what
we would expect, given the application that exists in the database, owned by the user
making the request.
"""